File: mobi_client/mobi_client/Assets/scripts/Config/ConfigModel/DatingGiftConfigModel.cs
namespace ConfigSystem
{
﻿    public class DatingGiftConfigModel :ConfigModel {
        public System.Collections.Generic.List<int> veryLike;//非常喜欢的礼物
        public string veryLikeResponeAvg;//非常喜欢的回应avg
        public System.Collections.Generic.List<int> like;//喜欢的礼物
        public string LikeResponeAvg;//喜欢的回应avg
        public string normalResponeAvg;//一般的回应avg
        public System.Collections.Generic.List<int> dislike;//不喜欢的礼物
        public string dislikeResponeAvg;//不喜欢的回应avg
        public System.Collections.Generic.List<int> returnGift;//回礼道具id
        public System.Collections.Generic.List<int> giftCount;//回礼道具数量
        public System.Collections.Generic.List<int> giftWeight;//回礼道具权重
        public string giftAvg;//回礼avg
    }
}

File: mobi_client/mobi_client/mobi_config/excel_check/temporary/java/DatingGiftConfigModel.java
package com.mobi.config.configModel;
import com.mobi.config.ConfigModel;
import com.mobi.config.wrappedArrayList.*;
public class DatingGiftConfigModel extends ConfigModel {
	public ArrInt veryLike;//非常喜欢的礼物
	public String veryLikeResponeAvg;//非常喜欢的回应avg
	public ArrInt like;//喜欢的礼物
	public String LikeResponeAvg;//喜欢的回应avg
	public String normalResponeAvg;//一般的回应avg
	public ArrInt dislike;//不喜欢的礼物
	public String dislikeResponeAvg;//不喜欢的回应avg
	public ArrInt returnGift;//回礼道具id
	public ArrInt giftCount;//回礼道具数量
	public ArrInt giftWeight;//回礼道具权重
	public String giftAvg;//回礼avg
}